test/src/org/labkey/test/tests/cds/CDSRReportsTest.java

```diff
@@ -158,7 +158,8 @@ public void validateViewReportComplex() throws MalformedURLException
         }
 
         goToProjectHome();
-        goToManageViews();
+        clickTab("Clinical and Assay Data");
+        waitAndClick(Locator.tagWithClass("span", "labkey-wp-icon-button-active").append(Locator.tagWithClass("span", "fa-pencil")));
```

Comment:
Is there a locator class that this can be added to?

Reply:
It's an odd situation for both CDS and LabKey. LabKey usually uses Manage Views, that one has a helper method. But edit from manage views doesn't provide user the ability to hide report. Click edit icon by web part name does provide user option to hide and that's what I need. So it's a one-off use case. CDS rarely tests outside the application so it's weird to add a LabKey location to CDS test page also.